code/src/components/app-badge/AppBadge.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { render, screen } from "@testing-library/react";
+import AppBadge from "./AppBadge";
+
+describe("AppBadge", () => {
+  it("renders AppBadge component", () => {
+    render(<AppBadge>children</AppBadge>);
+    const child = screen.getByText("children");
+    expect(child).toBeInTheDocument();
+  });
+
+  it("applies the no-round class when isRounded is false", () => {
+    const { container } = render(<AppBadge isRounded={false} />);
+    expect(container.firstChild).toHaveClass("spa-badge__no-round");
```

Comment:
Do we need to destructure `container` here? Maybe we can use just `screen.findByText`?

Reply:
fixed